lessons/09-clientside-frameworks/README.md

```diff
@@ -1,51 +1,57 @@
-You have tried doing this before, most notably in the burger ordering homework. 
-You probably noticed how messy that got for even a simple application. 
-You also know about APIs and how you can retrieve structured data and show it on your page. 
-Now consider a different structure in which almost your *entire server is simply an API* that sends data to a single page that changes appropriately. 
+You have tried doing this before, most notably in the burger ordering homework.
+You probably noticed how messy that got for even a simple application.
+You also know about APIs and how you can retrieve structured data and show it on your page.
+Now consider a different structure in which almost your *entire server is simply an API* that sends data to a single page that changes appropriately.
 In this structure the server would serve only a few different assets and include an API that can be used to request the necessary data.
-In that case the communications would look like this: 
+In that case the communications would look like this:
 ![ClientOrganizedSystem](images/clientsideWebCommunicationNew.png)
 
-This clearly has the benefit of almost certainly lowering the amount of data that is transferred between the server and the client, which also lowers the loading time when clicking on links. 
+This clearly has the benefit of almost certainly lowering the amount of data that is transferred between the server and the client, which also lowers the loading time when clicking on links.
 Another benefit of this application structure is that the API component can be used for many other purposes, such as mobile applications or even to other developers who want to expand on your websites functionality, as diagrammed below:
 ![UsesOfClientSide](images/apiDrivenDevelopmentNew.png)
 
-You may wonder why this is not how it has always been done. The simple answer to this is that it is only recently that Javascript, browsers, and computers have gotten good enough to reliably utilize these client-side frameworks. If you were going to do anything interesting, you had to do it on the server. In addition, this method may be more labor intensive for certain projects and can be a tricky adjustment to make. You will get to explore some of these strengths and weaknesses in the next lab assignment when you tackle one of these frameworks yourself. 
+You may wonder why this is not how it has always been done. The simple answer to this is that it is only recently that Javascript, browsers, and computers have gotten good enough to reliably utilize these client-side frameworks. If you were going to do anything interesting, you had to do it on the server. In addition, this method may be more labor intensive for certain projects and can be a tricky adjustment to make. You will get to explore some of these strengths and weaknesses in the next lab assignment when you tackle one of these frameworks yourself.
 
 ##Your First Client Templating
```

Comment:
Do we still want to do this? Is client side templating right before learning frameworks actually useful?